.github/ISSUE_TEMPLATE/1.bug_report.yml

```diff
@@ -0,0 +1,38 @@
+name: Bug Report
+description: Create a bug report for the JSS
+labels: ['🐞 bug']
+body:
+  - type: textarea
+    attributes:
+      label: Describe the Bug
+      description: Provide clear and concise description of the issue, and why you consider it to be a bug.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected Behavior

```

Comment:
I think the expected behavior should go after the scenario.
Also, do we need both "Describe the Bug" and "To Reproduce"? Could they be combined?

Reply:
Moved Expected Behavior after To Reproduce
Regarding Describe and To Reproduce, I think it's fine to leave them separately.
Description contains a context, To Reproduce - steps, screenshots etc, easier to separate them